test/e2e/virtual/apiexport/authorizer_test.go

```diff
@@ -98,131 +95,198 @@ func TestAPIExportAuthorizers(t *testing.T) {
-	// create API binding in tenant workspace pointing to the sherriffs export
-	apifixtures.BindToExport(ctx, t, serviceProvider1Path, "wild.wild.west", tenantPath, user3KcpClient)
-
-	t.Logf("Install today cowboys APIResourceSchema into service provider workspace %q", serviceProvider2Path)
-	user2serviceProvider2KcpClient, err := kcpclientset.NewForConfig(framework.UserConfig("user-2", rest.CopyConfig(cfg)))
-	require.NoError(t, err)
-	mapper := restmapper.NewDeferredDiscoveryRESTMapper(memory.NewMemCacheClient(user2serviceProvider2KcpClient.Cluster(serviceProvider2Path).Discovery()))
-	err = helpers.CreateResourceFromFS(ctx, user2DynamicClusterClient.Cluster(serviceProvider2Path), mapper, nil, "apiresourceschema_cowboys.yaml", testFiles)
-	require.NoError(t, err)
-
-	t.Logf("Create an APIExport for today cowboys APIResourceSchema in service provider %q", serviceProvider2Path)
-	cowboysAPIExport := &apisv1alpha1.APIExport{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "today-cowboys",
-		},
-		Spec: apisv1alpha1.APIExportSpec{
-			LatestResourceSchemas: []string{"today.cowboys.wildwest.dev"},
-			PermissionClaims: []apisv1alpha1.PermissionClaim{
-				{
-					GroupResource: apisv1alpha1.GroupResource{Group: "", Resource: "configmaps"},
-					All:           true,
+	t.Logf("Found identity hash: %v", sherriffsIdentityHash)
+
+	t.Logf("install cowboys API resource schema, API export, and permissions for user-3 to be able to bind to the export in service provider workspace %q", serviceProvider2Path)
+	require.NoError(t, apply(t, ctx, serviceProvider2Path, framework.UserConfig("user-2", rest.CopyConfig(cfg)),
+		&apisv1alpha1.APIResourceSchema{
+			ObjectMeta: metav1.ObjectMeta{Name: "today.cowboys.wildwest.dev"},
+			Spec: apisv1alpha1.APIResourceSchemaSpec{
+				Group: "wildwest.dev",
+				Names: apiextensionsv1.CustomResourceDefinitionNames{Plural: "cowboys", Singular: "cowboy", Kind: "Cowboy", ListKind: "CowboyList"},
+				Scope: "Namespaced",
+				Versions: []apisv1alpha1.APIResourceVersion{
+					{Name: "v1alpha1", Served: true, Storage: true, Schema: runtime.RawExtension{Raw: []byte(`{"type":"object"}`)}},
 				},
-				{
-					GroupResource: apisv1alpha1.GroupResource{Group: "wild.wild.west", Resource: "sheriffs"},
-					IdentityHash:  identityHash,
-					All:           true,
+			},
+		},
+		&apisv1alpha1.APIExport{
+			ObjectMeta: metav1.ObjectMeta{Name: "today-cowboys"},
+			Spec: apisv1alpha1.APIExportSpec{
+				LatestResourceSchemas:   []string{"today.cowboys.wildwest.dev"},
+				MaximalPermissionPolicy: &apisv1alpha1.MaximalPermissionPolicy{Local: &apisv1alpha1.LocalAPIExportPolicy{}},
+				PermissionClaims: []apisv1alpha1.PermissionClaim{
+					{
+						GroupResource: apisv1alpha1.GroupResource{Resource: "configmaps"},
+						ResourceSelector: []apisv1alpha1.ResourceSelector{
+							{Name: "claimed-cm", Namespace: "claimed-ns-by-serviceprovider-2"},
+							{Name: "claimed-deadlock-cm", Namespace: "deadlock-cm-namespace"},
```

Comment:
the above `t.Log()` do not explain what the intent is here

Reply:
oh this is a left-over from the reverse permission claims PR, good catch, removing.